pkg/controller/dataimportcron-conditions.go

```diff
@@ -88,13 +88,13 @@ func updateConditionState(condition *cdiv1.ConditionState, status corev1.Conditi
 	}
 }
 
-func updateDataImportCronOutdatedMetric(upToDateCondition *cdiv1.DataImportCronCondition, increment bool) {
+func updateDataImportCronOutdatedMetric(prevUpToDateCondition *cdiv1.DataImportCronCondition, increment bool) {
```

Comment:
Updated the parameter name here to reflect better that we're looking at the previous condition